provider/pkg/gen/dotnet-templates/helm/v2/Chart.cs

```diff
@@ -210,25 +222,366 @@ namespace Pulumi.Kubernetes.Helm.V2
     ///                     ["items"] = new Dictionary&lt;string, object&gt;(),
     ///                 }.ToImmutableDictionary();
     ///             }
-    /// 
+    ///
     ///             return obj;
     ///         }
     ///     }
     /// }
     /// ```
     /// </summary>
-    public sealed class Chart : ChartBase
+    public sealed class Chart : CollectionComponentResource
```

Comment:
Technically, that's a breaking change. Not sure if anyone uses the base class in their programs, but it's public.

Reply:
Oh, hmm. What's your suggestion given that the implementation has diverged between v2 and v3?

Reply:
We could keep the base class but nullify its code, in case somebody had references to it. It would still be breaking if somebody implemented another subclass, which I highly doubt.

Maybe better - we could keep the base class as-is for V2.Chart, but do not inherit from it in V3.Chart and document this in the changelog.

Reply:
> Maybe better - we could keep the base class as-is for V2.Chart, but do not inherit from it in V3.Chart and document this in the changelog.

Unfortunately, it looks like the v3 Chart class also inherited from `ChartBase`, so it would still potentially be a breaking change. Are you saying this is ok, and we should just document it in the changelog?

Reply:
Yes, that's what I suggested. As the change itself is about changing the behavior of the V3 chart, maybe it makes sense to take a slight break there. Just not seeing other great options.